e2e/integration/wordpress-01/__snapshots__/head-tags-aioseop-3.spec.ts.snap

```diff
@@ -1,117 +1,20 @@
-exports[`Head Tags - All in One SEO Pack > Post > should render the correct canonical link #0`] = `
+exports[`Head Tags - All in One SEO Pack 3 > Post > should render the correct canonical link #0`] = `
```

Comment:
I've modified the title passed to `describe(...)` in the spec file so the name of these snapshots has changed.